docker.sh

```diff
@@ -6,10 +6,6 @@ cd $(dirname $0)
 IMAGE=ownmailbox-image
 CONTAINER=ownmailbox
 
-docker() {
-    sudo docker "$@"
-}
-
```

Comment:
On Wed, Jul 20, 2016 at 6:47 PM, Armando Lüscher <notifications@github.com>
wrote:
>
> Why must docker be run as root?
>

Because otherwise it does not work, check out this:
https://docs.docker.com/engine/installation/linux/ubuntulinux/#/create-a-docker-group
I don't know about OSX.